src/molecules/Input/Input.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Inputfield, Label } from "../../atoms";
+import { ICustomInputProps } from "../../atoms/InputField";
+import { ICustomLabelProps } from "../../atoms/InputLabel";
+import "./input.scss";
+
+const Input = (
+  props: React.PropsWithChildren<ICustomInputProps & ICustomLabelProps>
+): React.ReactElement => {
+  return (
+    <div>
+      <Label {...props} />
+      <Inputfield {...props} />
+      <div className="message">
```

Comment:
Do we need to render two whole elements if there's nothing inside of them? If we do for the layout then that's fine~